apps/learner/src/routes/(app)/+page.svelte

```diff
@@ -37,13 +38,24 @@
       onplay={handlePlay}
     />
 
-    {#if isFloatingPlayerVisible}
-      <FloatingPlayer
-        to="/"
-        title="Navigating Special Educational Needs in Singapore: A Path to Inclusion"
-        isplaying={isPlaying}
-        onplay={togglePlayPause}
-      />
-    {/if}
+    <div class="z-100 pointer-events-none fixed inset-x-0 bottom-0">
```

Comment:
Can try move the entire floating container out of this page container?

Reply:
I think I move it to the layout for my task.